src/ape/api/projects.py

```diff
@@ -157,7 +160,7 @@ def _create_compilers_list(
                     Compiler(
                         name=compiler.name,
                         version=str(version),
-                        settings={},
+                        settings=settings,
```

Comment:
This should be like the `version_settings`, not the full settings lists for each compiler. Get the settings for _this_ version.